tests/wrap_command.bats

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bats
+
+load '/usr/local/lib/bats/load.bash'
+load '../lib/shared'
+load '../lib/run'
+load '../lib/wrap_command'
+
+# export DOCKER_COMPOSE_STUB_DEBUG=/dev/tty
```

Comment:
I'm guessing this is for debugging?

Reply:
Yes, all of the specs in this repo keep those lines for debugging failures.